src/app/teams/teams-member.component.ts

```diff
@@ -34,6 +35,7 @@ export class TeamsMemberComponent implements OnInit {
 
   ngOnInit() {
     this.memberType = this.member.teamId === undefined ? 'community' : 'other';
+    this.leadershipTitleType = this.leadershipTitleType === undefined ? 'Add' : this.leadershipTitleType === '' ? 'Add' : 'Change';
```

Comment:
This only will set the text when the component initializes.  You should use `ngOnChanges` so the text updates after a title is added.

Also, it is cleaner to use a different property local to the component for the `'Add'`/`'Change'` value.  If another one of the input properties changes, the text would change from **Add** to **Change** because the value of `leadershipTitleType` is now `'Add'`.